### ISSUE TITLE
About the loss at the time of training

### ISSUE THREAD

**Issue by wingerboy** [Stinky-Tofu/YOLO_v2#1]
Firstly, thank you for your open source code.
I use your model to train my own training set, and the loss can only be reduced to around 0.7. But I trained with darknet and was able to drop to around 0.1. How can I adjust?

**Comment by Stinky-Tofu** [comment]
This project is deprecated. Because it has many problems (training method, loss function, data augmentation, multi-scale training, etc.), which I have not solved so far, so it can only get 45% mAP on the VOC2007 dataset. Please see YOLO3 https://github.com/Stinky-Tofu/YOLO_V3, which get 85% mAP on the VOC2007 dataset, and I will continue to improve YOLO3 with the latest papers.

**Comment by YaoLing13** [comment]
Hello,Thank you for your code, it helps me a lot, and i encountered some problems.
When i train dataset of VOC, the loss train dataset is 0.0223 and it has been decreasing, but the loss of valid dataset is 7.2002. And i test the model, it output nothing!  Haved you encountered such a problem,and how big is your epoch setting? Than you for your answer.

**Comment by Stinky-Tofu** [comment]
> Hello,Thank you for your code, it helps me a lot, and i encountered some problems.
> When i train dataset of VOC, the loss train dataset is 0.0223 and it has been decreasing, but the loss of valid dataset is 7.2002. And i test the model, it output nothing! Haved you encountered such a problem,and how big is your epoch setting? Than you for your answer.

Sorry... this project is deprecated. Because it has many problems (training method, loss function, data augmentation, multi-scale training, etc.), which I have not solved so far, so it can only get 45% mAP on the VOC2007 dataset. Please see YOLO3 https://github.com/Stinky-Tofu/YOLO_V3, which get 85% mAP on the VOC2007 dataset, and I will continue to improve YOLO3 with the latest papers.

**Comment by YaoLing13** [comment]
Thank you
